### PR TITLE
INDY-833: Init bls stewards

### PR DESCRIPTION
**Pull request by ashcherbakov**
- Add a script to enable BLS for Steward
- Improved NODE txn tests

### CODE REVIEW

File: indy_client/test/cli/test_node.py [hunk 1]
```diff
@@ -7,59 +7,89 @@
...
+                                     newStewardCli, newNodeVals,
+                                     newNodeAdded):
     be(newStewardCli)
     newNodeVals['newNodeData'][ALIAS] = randomString(6)
     doSendNodeCmd(do, newNodeVals,
                   expMsgs=['existing data has conflicts with request data'])
     exitFromCli(do)
+
+
+def test_update_alias(be, do,
```
Comment (mzk-vct) [inline indy_client/test/cli/test_node.py]: Please add description for this test

File: setup.py [hunk 2]
```diff
@@ -119,7 +119,8 @@ def run(self):
              'scripts/test_some_write_keys_others_read_them',
              'scripts/test_users_write_and_read_own_keys',
              'scripts/validator-info',
-             'scripts/init_bls_keys'],
+             'scripts/init_bls_keys',
+             'scripts/init_bls_for_stewards'],
```
Comment (mzk-vct) [inline setup.py]: This name is confusing.
As far as I understand this script creates bls keys and sends transaction to publish them, but it's name tells that it adds bls keys for steward (as if steward can have bls keys)

File: scripts/init_bls_for_stewards [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Generate BLS keys for a node "
+                    "by taking the node's name and seeds "
+                    "and send NODE txn with the BLS key specified")
+
+    parser.add_argument('--name', required=True, help='node name')
+    parser.add_argument('--node_seed', required=True, type=str,
```
Comment (mzk-vct) [inline scripts/init_bls_for_stewards]: If node_seed is used only for generation of `dest` could you please add option to specify `dest` directly?

Comment (ashcherbakov) [reply scripts/init_bls_for_stewards]: The requirements for the task is that BLS key needs to be created for a seed (the same seed as used for node keys actually). So I think this option is not required (let's add it if this requirement appears)

File: setup.py (hunk 2 above)
Comment (ashcherbakov) [reply setup.py]: Renamed to `enable_bls`

File: indy_client/test/cli/test_node.py (hunk 1 above)
Comment (ashcherbakov) [reply indy_client/test/cli/test_node.py]: Done

**Pull request merged**